gravitee-apim-console-webui/src/management/api/import-v4/api-import-v4.component.ts

```diff
@@ -43,11 +45,25 @@ import { ApiV4 } from '../../../entities/management-api-v2';
     ReactiveFormsModule,
     RouterModule,
     ApiImportFilePickerComponent,
+    MatCheckbox,
+    GioFormSlideToggleModule,
+    MatSlideToggle,
   ],
   templateUrl: './api-import-v4.component.html',
   styleUrl: './api-import-v4.component.scss',
 })
-export class ApiImportV4Component implements AfterViewInit {
+export class ApiImportV4Component implements AfterViewInit, OnInit {
+  ngOnInit(): void {
+    this.form.controls['format'].valueChanges.subscribe((value) => {
```

Comment:
can you add an unsubscription on this subscribe. we try to do it for all front subscribe.
With `.pipe(takeUntil(this.unsubscribe$))` (to check example from others component)